zone_test.go

```diff
@@ -17,3 +18,28 @@ func TestSizeofZone(t *testing.T) {
 	var z Zone
 	t.Logf("sizeof Zone = %d", unsafe.Sizeof(z))
 }
+
+func TestLanguages(t *testing.T) {
+	tests := []struct {
+		domain string
+		want   []string
+	}{
+		{"com", nil},
+		{"中国", []string{"zh-Hans-CN"}},
+		{"中國", []string{"zh-Hant-CN"}},
+		{"台灣", []string{"zh-Hant-TW"}},
+		{"台灣", []string{"zh-Hant-TW"}},
```

Comment:
This appears to be identical to the one above

Reply:
Fixed, thanks!